### PR TITLE
remove operation id prefix

### LINKED ISSUE #866: Requires remove operationid prefix
When use smallrye openapi operationid strategy CLASS_METHOD,operationId fields take the form of "class_name_method". For example, "User_findAll".
we need remove operationid prefix that already exists in the OpenAPI Generator.

### PR DESCRIPTION
**Pull request by yuhaibohotmail**
- [x]  You have read the [contributors guide](CONTRIBUTING.md)
- [x]  Your code is properly formatted according to [our code style](CONTRIBUTING.md#code-style)
- [x]  Pull Request title contains the target branch if not targeting main: `[0.9.x] Subject`
- [x] Pull Request contains link to the issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x]  Pull Request does not include fixes for issues other than the main ticket

Fix #866

When use smallrye openapi operationid strategy CLASS_METHOD,operationId fields take the form of "class_name_method". For example, "User_findAll". We want remove operationId prefix (e.g. User_findAll=> findAll)

In this PR, we add a config properties remove-operation-id-prefix,to delete class name from operationId.  The properties are passed to the openapi-generator, and the specific functions are executed by the openapi-generator.

https://github.com/OpenAPITools/openapi-generator/blob/4c5a57fe7bae59483bb8fa9218be8dc4f82e1bda/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L5638

### CODE REVIEW

**Comment by yuhaibohotmail** [comment]
@ricardozanini   @ricardozanini 
Sorry I messed up my commit, so I rebuilt this PR.

**Review by ricardozanini — APPROVED** [review]
It's an unusual use case; many thanks for bringing this to our attention and sending the PR. 

Now, with the test, it makes much more sense.

File: client/integration-tests/remove-operationid-prefix/src/test/java/io/quarkiverse/openapi/generator/it/RemoveOperatiionIdPrefixTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package io.quarkiverse.openapi.generator.it;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class RemoveOperatiionIdPrefixTest {
```
Comment (hbelmiro) [inline client/integration-tests/remove-operationid-prefix/src/test/java/io/quarkiverse/openapi/generator/it/RemoveOperatiionIdPrefixTest.java:10]: Typo.

```suggestion
public class RemoveOperationIdPrefixTest {
```

File: client/integration-tests/remove-operationid-prefix/src/test/java/io/quarkiverse/openapi/generator/it/RemoveOperatiionIdPrefixTest.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    String apiClassName = "org.openapi.quarkus.openapi_remove_operation_id_prefix_yaml.api.UserResourceApi";
+    String modelClassName = "org.openapi.quarkus.openapi_remove_operation_id_prefix_yaml.model.User";
+
+    @Test
+    void apiIsBeingGenerated() throws NoSuchMethodException {
```
Comment (hbelmiro) [inline client/integration-tests/remove-operationid-prefix/src/test/java/io/quarkiverse/openapi/generator/it/RemoveOperatiionIdPrefixTest.java:16]:
```suggestion
    void apiIsBeingGenerated() {
```

**Review by hbelmiro — CHANGES_REQUESTED** [review]

**Review by hbelmiro — APPROVED** [review]

**Review by hbelmiro — APPROVED** [review]

**Pull request merged**